cg/meta/meta.py

```diff
@@ -10,7 +10,6 @@
 from cg.apps.tb import TrailblazerAPI
 from cg.apps.vogue import VogueAPI
 from cg.meta.compress import CompressAPI
-from cg.meta.upload.gisaid import GisaidAPI
```

Comment:
I wonder if we should add this to the init instead of removing it. Maybe @Mropat or @mayabrandi knows

Reply:
It was unused so I removed it, true, hopefully they know better!